angular/projects/spark-angular/src/lib/components/sprk-alert/sprk-alert.component.spec.ts

```diff
@@ -28,74 +38,165 @@ describe('SprkAlertComponent', () => {
   it('getClassesAlertContainer should match what gets set on the container', () => {
     fixture.detectChanges();
     expect(alertElement.classList.toString()).toEqual(
-      component.getClassesAlertContainer()
+      component.getClassesAlertContainer(),
     );
   });
 
-  it('should add the correct class if alertType is not set', () => {
-    component.alertType = '';
+  it('should default to info alertType if alertType and variant are not set', () => {
     fixture.detectChanges();
-    expect(component.getClassesAlertContainer()).toEqual('sprk-c-Alert');
+    expect(component.variant).toEqual(undefined);
+    expect(component.alertType).toEqual('info');
+    expect(component.getClassesAlertContainer()).toEqual(
+      'sprk-c-Alert sprk-c-Alert--info',
+    );
+    expect(iconElement.querySelector('use').getAttribute('xlink:href')).toEqual(
+      '#bell-filled',
+    );
+  });
+
+  it('should prefer variant if both variant and alertType are set', () => {
+    component.alertType = 'success';
+    component.variant = 'fail';
+    fixture.detectChanges();
+    expect(alertElement.classList.contains('sprk-c-Alert--success')).toEqual(
```

Comment:
How come we are checking the classes on the element here and the classes in the method in the following line vs. both one or the other?

Reply:
I wanted to add a check that the success class wasn't being added because the `alertType` was being set to success. I can remove it,

Reply:
I think its a great test! I was curious about the difference in checks. With one using the element and one using the method to check.

Reply:
Okay, thats what I was curious about!